internal/monitors/haproxy/metadata.yaml

```diff
@@ -0,0 +1,767 @@
+monitors:
+  - monitorType: haproxy
+    doc: |
+      This monitor scrapes [HAProxy](http://www.haproxy.org/) statistics from an HTTP endpoint or a UNIX socket.
+      It requires HAProxy 1.8+. All metrics have a `process_num` dimension which is the index of the HAProxy
+      process that the metric pertains to.
```

Comment:
was: All metrics have a `process_num` dimension which is the index of the HAProxy
      process that the metric pertains to.
change to: The HAProxy process each metric pertains to is indicated by the metric dimension `process_num`.

Reply:
ok